Dangerfile

```diff
@@ -95,6 +99,9 @@ warn("Big PR -- consider splitting it up into multiple changesets") if git.lines
 # License is immutable
 fail("Do not modify the License") if @S_LICENSE_FILES && checkFiles(@S_LICENSE_FILES).any?
 
+# Notify about modifications to files that we've backported explicitly
+warn("This change includes modification to a file that was backported from newer Django.") if @S_BACKPORTED_FILES && checkFiles(@S_BACKPORTED_FILES).any?
```

Comment:
Looking forward to us all editing this file after we inevitably `rm -rf` Danger. 😜